packages/dedupe-mixin/README.md

```diff
@@ -0,0 +1,216 @@
+# Dedupe Mixin
+
+[//]: # 'AUTO INSERT HEADER PREPUBLISH'
+
+Making sure your Mixin does not interfere with itself.
+
+## Features
+
+- small
+- fast
+- typed
+
+## Usage
+
+Apply it to all mixins that are shared.
+
+```js
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+
+export const MyMixin = dedupeMixin(
+  superclass =>
+    class MyMixin extends superclass {
+      // your mixin code goes here
+    },
+);
+```
+
+## What is a Mixin?
```

Comment:
Put this before the "Usage" section

Reply:
not sure about this one... 
what is if people already know what a mixin is and they only come here to find out how to dedupe it?
so it's basically how many people with know mixins vs how many will not?

if less then 50% know mixins then it should be above - otherwise "as details below"